apps/frontend/src/app/pages/person/create-user-profile/create-user-profile.component.html

```diff
@@ -0,0 +1,37 @@
+@if(step === 1) {
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/b9d81a3e-cbf1-4a2a-907a-74dce7ba51f1)

When I finish the completion of the steps, at the end, the data I filled isn't being displayed